papyri/tests/test_parse.py

```diff
@@ -1,9 +1,16 @@
 from textwrap import dedent
+from pathlib import Path
 
 import pytest
 
 from papyri import errors
-from papyri.ts import parse
+from papyri.ts import parse, Node, TSVisitor
+from tree_sitter import Language, Parser
+
+parser = Parser()
+pth = str(Path(__file__).parent.parent / "rst.so")
+RST = Language(pth, "rst")
+parser.set_language(RST)
```

Comment:
can't we just import parser from papyri.ts ?

Reply:
Probably 😁 will fix